src/coprocessor/endpoint.rs

```diff
@@ -73,27 +74,36 @@ pub struct Host {
     sched: Scheduler<Task>,
     reqs: HashMap<u64, Vec<RequestTask>>,
     last_req_id: u64,
-    pool: ThreadPool<SmallGroupFirstQueue<u64>, u64>,
+    pool: ThreadPool<FifoQueue<u64>, u64>,
+    low_priority_pool: ThreadPool<FifoQueue<u64>, u64>,
+    high_priority_pool: ThreadPool<FifoQueue<u64>, u64>,
     max_running_task_count: usize,
 }
 
 impl Host {
-    pub fn new(engine: Box<Engine>,
-               scheduler: Scheduler<Task>,
-               concurrency: usize,
-               txn_concurrency_on_busy: usize,
-               small_txn_tasks_limit: usize)
-               -> Host {
-        let queue = SmallGroupFirstQueue::new(txn_concurrency_on_busy, small_txn_tasks_limit);
+    pub fn new(engine: Box<Engine>, scheduler: Scheduler<Task>, concurrency: usize) -> Host {
         Host {
             engine: engine,
             sched: scheduler,
             reqs: HashMap::default(),
             last_req_id: 0,
             max_running_task_count: DEFAULT_MAX_RUNNING_TASK_COUNT,
-            pool: ThreadPool::new(thd_name!("endpoint-pool"), concurrency, queue),
+            pool: ThreadPool::new(thd_name!("endpoint-normal-pool"),
```

Comment:
the thread name may be too long and can't be fully shown in `top` and flamegraph.